packages/amplify-codegen/src/utils/getCodegenPackageName.js

```diff
@@ -0,0 +1,12 @@
+const CODEGEN_OLD_PACKAGE = 'amplify-codegen-appsync-model-plugin';
+const CODEGEN_NEW_PACKAGE = '@aws-amplify/app-sync-model-gen-plugin';
```

Comment:
Typo: new package should be '@aws-amplify/appsync-modelgen-plugin'

Reply:
done